src/pages/modeler/index.hbs

```diff
@@ -27,7 +27,8 @@ layout: 'base.hbs'
 
 <style>
   .thumbnail {
-    padding: 0
+    padding: 0;
+    overflow: hidden;
```

Comment:
There was a nasty overflow of the thumbnails over their rounded borders, so I added this

Reply:
:+1: Always in for fixing these little glitches :tada: